VERSION

```diff
@@ -1 +1 @@
-0.24.0-dev
+0.24.0
```

Comment:
I think this now needs to be `0.25.0-dev`?

Reply:
Ach yes that's right

Reply:
:tada: done